posts/v2-metrics.md

```diff
@@ -26,7 +26,7 @@ So from this we can say that the branding has accounted for approximately a doub
 
 The second thing we wanted to improve was the amount of mentoring per solution. In V1 it was a bit of a lottery - you might get comments or you might not, purely dependent on whether someone noticed your submission. A big part of the formal mentoring of V2 was about making this fairer, consistent and reliable.
 
-The metric I've used here is what per of solutions received a comment.
+The metric I've used here is _comments per solution_.
```

Comment:
This is a different metric. Perhaps "percentage of solutions that received one or more comments" or something like that?

Reply:
I wasn't sure about this either.  Closest thing I immediately interpreted the original statement to.  @iHiD ?

Reply:
"what per" I almost translated to "what percent" but that might have been reading too much into what is _not_ an abbreviation.